.gitignore

```diff
@@ -7,3 +7,4 @@ deb_dist/*
 *.iml
 *.pyc
 .tox
+.eggs
```

Comment:
Should not be the case as it's a directory where things downloaded by `setuptools` are placed, plus the build is green with all test passing.